bin/jobsub_fetchlog

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python3 -I
+
+#
+# jobsub_fetchlog -- tool for downloading job output files from condor
+# COPYRIGHT 2022 FERMI NATIONAL ACCELERATOR LABORATORY
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+#
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import os
+import os.path
+import subprocess
+import shutil
+
+import htcondor
+
+#
+# we are in prefix/bin/jobsub_fetchlog, so find our prefix
+#
+PREFIX = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+#
+# find parts we need in package management
+#
+sys.path.append(os.path.join(PREFIX, "lib"))
+
+#
+# import our local parts
+#
+import creds
+from get_parser import StoreGroupinEnvironment
+from condor import Job
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-J", "--jobid", help="job/submission ID", required=True)
+    parser.add_argument(
+        "--destdir",
+        "--dest-dir",
+        "--unzipdir",
+        help="Directory to automatically unarchive logs into",
+    )
+    parser.add_argument(
+        "--archive-format",
+        help='format for downloaded archive: "tar" (default, compressed with gzip) or "zip"',
+        default="tar",
+    )
+    parser.add_argument(
+        "-G",
+        "--group",
+        help="Group/Experiment/Subgroup for priorities and accounting",
+        action=StoreGroupinEnvironment,
+        default=os.environ.get("GROUP", None),
+    )
+    parser.add_argument(
+        "--verbose",
+        action="store_true",
+        default=False,
+        help="dump internal state of program (useful for debugging)",
+    )
+
+    return parser
+
+
+def main():
+    """script mainline:
+    - parse args
+    - get credentials
+    - get job info
+    - condor_transfer_data
+    - make tarball
+    """
+    parser = get_parser()
+    args = parser.parse_args()
+
+    if args.verbose:
+        htcondor.set_subsystem("TOOL")
+        htcondor.param["TOOL_DEBUG"] = "D_FULLDEBUG"
+        htcondor.enable_debug()
```

Comment:
This can be a bit chatty, but I think we should consider setting these options for all commands when the verbose/debug flag is set, otherwise the condor commands can be black holes.

Reply:
Agreed, and I’ll open up another issue to clean up some of the too-verbose output that otherwise appears, so that there’s little noise without verbose, and lots of info with,